Atlassian.Jira/Remote/IssueLinkService.cs

```diff
@@ -71,14 +71,23 @@ public async Task<IEnumerable<IssueLink>> GetLinksForIssueAsync(Issue issue, IEn
                 issuesMap.Add(issue.Key.ToString(), issue);
             }
 
+            var receivedLinksCount = filteredIssueLinks.Count();
+
+            filteredIssueLinks = filteredIssueLinks.Where(issueLink =>
+            {
+                var (outwardIssueKey, inwardIssueKey) = GetLinkedIssueKeysFromJObject(issueLink, string.Empty);
+                return issuesMap.ContainsKey(outwardIssueKey) || issuesMap.ContainsKey(inwardIssueKey);
+            });
+
+            if (receivedLinksCount > filteredIssueLinks.Count())
+            {
+                Console.WriteLine($"We're missing {receivedLinksCount - filteredIssueLinks.Count()} issues, probably because they're archived");

```

Comment:
they don't have any proper logger?

Reply:
They don't use any in the project, as far as I could tell.